packages/terra-status-view/src/terra-dev-site/test/status-view/StatusViewThinConstraint.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Base from 'terra-base';
+import Button from 'terra-button';
+import StatusView from '../../../StatusView';
+
+const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
+const ipsum = 'Lorem ipsum dolor';
+
+export default () =>
+  <Base locale={locale}>
```

Comment:
Lets drop the base wrapper. Before terra-dev-site, each test page was standalone from the site which meant they needed to supply their own base wrapper. terra-dev-site now provides this for us! (yay). This is code cleanup thats needs to happen at some point, but haven't quite gotten there.